src/zfm20/zfm20.cxx

```diff
@@ -540,10 +439,10 @@ uint8_t ZFM20::deleteDB()
   return rPkt[9];
 }
 
-uint8_t ZFM20::search(int slot, uint16_t *id, uint16_t *score)
+uint8_t ZFM20::search(int slot, uint16_t &id, uint16_t &score)
 {
```

Comment:
I see - hadn't paid attention to the interface files :)  

In addition to documentation, there's also the issue that the API is significantly different between C++ and SWIG languages...